Controller/HelperController.php

```diff
@@ -110,7 +110,7 @@ public function appendFormFieldElementAction(Request $request)
         // render the widget
         // todo : fix this, the twig environment variable is not set inside the extension ...
 
-        $extension = $this->twig->getExtension('form');
+        $extension = $this->twig->getExtension('Symfony\Bridge\Twig\Extension\FormExtension');
```

Comment:
Ok I'm back! Have a look at [this line](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/Tests/Controller/HelperControllerTest.php#L327). The extension is added to the environment, but it is not an instance of Symfony\Bridge\Twig\Extension\FormExtension, it's a mock of `\Twig_ExtensionInterface`. Solution: don't use a mock, us the real class, which should be maybe difficult but will be better b/c you're not really supposed to mock what you don't own.